aiopenapi3/v30/glue.py

```diff
@@ -424,19 +425,18 @@ def _prepare(self, data: Optional["RequestData"], parameters: Optional["RequestP
         self._prepare_body(data, rbq)
 
     def _process__status_code(self, result: httpx.Response, status_code: str) -> "v3xResponseType":
-        # find the response model in spec we received
-        expected_response = None
-        if status_code in self.operation.responses:
-            expected_response = self.operation.responses[status_code]
-        elif "default" in self.operation.responses:
-            expected_response = self.operation.responses["default"]
-
+        expected_response = (
+            self.operation.responses.get(status_code)
+            or self.operation.responses.get(status_code[0] + "XX")
+            or self.operation.responses.get("default")
+        )
         if expected_response is None:
             options = ",".join(self.operation.responses.keys())
             raise HTTPStatusError(
                 self.operation,
                 result.status_code,
-                f"""Unexpected response {result.status_code} from {self.operation.operationId} (expected one of {options}), no default is defined""",
+                f"Unexpected response {result.status_code} from {self.operation.operationId} "
+                f"(expected one of {options}), no default is defined",
                 result,
```

Comment:
using line length 80?

Reply:
Ahh, seems so. My preferred formatter is black and 80 char lines is default. I must have auto-formatted while testing in a Jupyter notebook before copying the code over.

Reply:
I use pre-commit, black is part of it, can recommend.